ui/main_window.cpp

```diff
@@ -694,6 +697,13 @@ void MainWindow::OnAbout()
     about->exec();
 }
 
+//--------------------------------------------------------------------------------------------------
+void MainWindow::OnShortcuts()
+{
+    ShortcutsDialog *shortcuts = new ShortcutsDialog();
```

Comment:
do you need to manually delete `shortcuts`? from what i understand, if the QObjects doesn't have a parent, we need to manually delete the pointer?

Reply:
I don't think I do since to close any QDialog, the 'ESC' key or the "close" or 'x' buttons need to be clicked. Doing either of those results in a QCloseEvent() and since the dialog is created with 'exec' Qt automatically deletes it when it is closed. From my understanding, if show() was used instead, then the dialog would need to be explicitly deleted.